In the console, type clear and hit Enter or Return. 

This will clear the console output. 

Run the file once while the tasks are longer. 

Copy and paste the terminal results (when task_duration is 3) here.

STARTING UP............................. 
It's 2023-01-23 at 05:16 PM

This file is running on:    nt Windows 10
The Python version is:      3.10.9       

The Python interpreter is at:
 C:\Users\dills\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\python.exe


DELETED tabble pets.
CREATED table pets.

P1 getting ready to insert Ace the Dog.
P1 ADDED Ace the Dog.
P2 getting ready to insert Cooper the Rabbit.
P3 getting ready to insert Emma the Rabbit.
P1 getting ready to insert Buddy the Dog.
P1 ADDED Buddy the Dog.
Process Process-2:
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\multiprocessing\process.py", line 314, in _bootstrap    
    self.run()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "c:\Users\dills\OneDrive\Documents\streaming-02-multiple-processes\multiple_processes.py", line 106, in process_two
    insert_pet("P2", "Cooper", "Rabbit")
  File "c:\Users\dills\OneDrive\Documents\streaming-02-multiple-processes\multiple_processes.py", line 95, in insert_pet
    cur.execute(sql)
sqlite3.OperationalError: database is locked
Process Process-3:
Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\multiprocessing\process.py", line 314, in _bootstrap    
    self.run()
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.10_3.10.2544.0_x64__qbz5n2kfra8p0\lib\multiprocessing\process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "c:\Users\dills\OneDrive\Documents\streaming-02-multiple-processes\multiple_processes.py", line 110, in process_three
    insert_pet("P3","Emma", "Rabbit")
  File "c:\Users\dills\OneDrive\Documents\streaming-02-multiple-processes\multiple_processes.py", line 95, in insert_pet
    cur.execute(sql)
sqlite3.OperationalError: database is locked